Source/santad/Logs/EndpointSecurity/Serializers/Protobuf.mm

```diff
@@ -17,7 +17,7 @@
 #include <EndpointSecurity/EndpointSecurity.h>
 #include <Kernel/kern/cs_blobs.h>
 #include <bsm/libbsm.h>
-#include <google/protobuf/arena.h>
+// #include <google/protobuf/arena.h>
```

Comment:
nit: uncomment?

Reply:
Whoops, moved to header. Fixed. (also fixed an unrelated header that had the same problem - `SNTNotificationManagerTest.m`)